src/framework/test_group.ts

```diff
@@ -138,13 +138,16 @@ class RunCaseSpecific<F extends Fixture> implements RunCase {
 
     try {
       const inst = new this.fixture(rec, this.params || {});
+      // Note: init() may throw SkipTestCase, in which case finalize() is not run.
```

Comment:
Hmm.. but why not have `finalize()` in a `finally` block?

I guess right now there's no need for it and we can add it in the future if necessary.

Reply:
I was thinking it doesn't make sense to finalize if init fails, but maybe it does. The only thing that would happen right now (I think) is that the popErrorScopes would fail.